docker/docker-compose.yml

```diff
@@ -4,23 +4,17 @@ services:
   # Mainflux services
   users:
     image: mainflux/users:0.10.0
-<<<<<<< HEAD
-    env_file: '../../stack/env.d/mainflux-users.env'
-=======
     env_file: './env.d/mainflux-users.env'
->>>>>>> c49e63fc801cab77aac739ca27144dd38a912c85
+    ports:
+      - 8180:8180
```

Comment:
I think you could amend those fixes to the previous commit

Reply:
Done. I removed them in the commit [10dcbfd](https://github.com/CESARBR/knot-cloud-storage/commit/10dcbfd55691901be7c024fc35ac3fcf4419b2ec).